Box.V2/Models/BoxGroupFolderCollaborationEventSource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+
+namespace Box.V2.Models
+{
+    public class BoxGroupFolderCollaborationEventSource : BoxEntity
+    {
+        public const string FieldFolderId = "folder_id";
+        public const string FieldFolderName = "folder_name";
+        public const string FieldGroupId = "group_id";
+        public const string FieldGroupName = "group_name";
+        public const string FieldParent = "parent";
+
+        [JsonProperty(PropertyName = FieldFolderId)]
+        new public string FolderId { get; private set; }
```

Comment:
You shouldn't need `new` on any of these properties, since they doesn't collide with any properties in the base class.